python/nano/setup.py

```diff
@@ -144,6 +144,13 @@ def setup_package():
                         "packaging",
                         "sigfig",
                         "setuptools<66"]
+    
+    # diffusion requires
+    diffusion_requires = pytorch_requires + [
+        "diffusers==0.14.0",
+        "transformers",
+        "openvino-dev==2022.2.0"
```

Comment:
remove openvino dependency. By default install option we support only pytorch 113, linux, ipex